src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\LanguageConstruct;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\AllowedValueSubset;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\VersionSpecification;
+use PhpCsFixer\FixerDefinition\VersionSpecificCodeSample;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ */
+final class SingleSpaceAfterConstructFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
+{
+    /**
+     * @var array
+     */
+    private static $tokenMap = [
+        'break' => T_BREAK,
+        'clone' => T_CLONE,
+        'continue' => T_CONTINUE,
+        'echo' => T_ECHO,
+        'goto' => T_GOTO,
+        'include' => T_INCLUDE,
+        'include_once' => T_INCLUDE_ONCE,
+        'new' => T_NEW,
+        'print' => T_PRINT,
+        'require' => T_REQUIRE,
+        'require_once' => T_REQUIRE_ONCE,
+        'return' => T_RETURN,
+        'throw' => T_THROW,
+        'yield' => T_YIELD,
+    ];
+
+    /**
+     * @var array
+     */
+    private $fixTokenMap = [];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        if (70000 <= \PHP_VERSION_ID && \defined('T_YIELD_FROM')) {
+            self::$tokenMap = array_merge(self::$tokenMap, [
```

Comment:
These are both "hardcoded" arrays. Doesn't it make more sense to directly add the values, instead of `array_merge`?